submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,17 @@
+### Complexity Corner: The Mall <author>The Kool</author>
+
+Welcome back once more to the Complexity Corner.  Today I'll be exploring a technique instead of a technology: the Mall.  As always with these articles, we'll be asking three questions: 'What is it?', 'What results can it give?', and 'Where should it be used?'
+
```

Comment:
Could you add a collage/gallery of maybe 4-5 different malls to show off how different they can be and what can be included?